testFile.hs

```diff
@@ -0,0 +1,8 @@
+module ParseStuff where 
+
+parseStuff :: String -> String
+parseStuff s = case s of 
```

Comment:
What is the purpose of this file?

Reply:
It was a test file I tried to commit and push to the master branch before I realised I shouldn't do that.